README.md

```diff
@@ -229,6 +229,13 @@ Run frontend tests:
 
 Cypress integration tests are implemented in [Cypress.io](https://www.cypress.io/).
 
+To install Cypress:
+
+```
+cd frontend
+yarn run cypress install
```

Comment:
[`yarn install`](https://github.com/openshift/console/pull/13318/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R183) already covers this, no?

Reply:
Unfortunately, no, commands like `yarn run test-cypress-pipelines` need a chrome installation. At least on Linux I can confirm that this step is required. (Unsure if the headless mode does this automatically... :man_shrugging:)

Reply:
Note I changed all Cypress configs to use Electron (which is built in to Cypress) as the default browser with https://github.com/openshift/console/pull/13300 to normalize how the tests are running locally vs. CI.  Cypress is definitely getting installed with `yarn install` and running the tests in Electron, so I don't think this README update is necessary.

Reply:
Actually, yesterday, when I was trying to run the e2e tests after pulling from the latest master, I was getting the error stating the new version of cypress used in the application, was not found in my system. I rebuilt the frontend, but still, the problem persisted. Then, Christoph suggested running the above command, which would install the latest Cypress version of the application in my system. After running it, it worked as expected.

Therefore, I feel this command will come in handy in case of Cypress upgrades in the future.